features/attach_validtoken.feature

```diff
@@ -494,3 +494,73 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
           | xenial  | disabled |
           | bionic  | disabled |
           | focal   | n/a      |
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Attach and Check for contract change in status checking
+       Given a `<release>` machine with ubuntu-advantage-tools installed
+       When I attach `contract_token` with sudo
+       Then stdout matches regexp:
+       """
+       UA Infra: ESM enabled
+       """
+       And stdout matches regexp:
+       """
+       This machine is now attached to
+       """
+       And stdout matches regexp:
+       """
+       SERVICE       +ENTITLED  STATUS    DESCRIPTION
+       cc-eal       +yes      +<cc_status>     +Common Criteria EAL2 Provisioning Packages
+       """
+       And stdout matches regexp:
+       """
+       esm-infra    +yes      +enabled  +UA Infra: Extended Security Maintenance \(ESM\)
+       fips         +yes      +<fips_status> +NIST-certified core packages
+       fips-updates +yes      +<fips_status> +NIST-certified core packages with priority security updates
+       livepatch    +yes      +n/a  +Canonical Livepatch service
+       """
```

Comment:
Lets keep this simpler since this isn't the main intention of this test
```suggestion
       And stdout matches regexp:
       """
       esm-infra    +yes      +enabled  +UA Infra: Extended Security Maintenance \(ESM\)
       """
```

Reply:
Makes sense. I always thought that is some boilerplate that should always be there 😄😄